lib/polars/lazy_frame.rb

```diff
@@ -38,6 +38,7 @@ def self._scan_csv(
       dtypes: nil,
       null_values: nil,
       ignore_errors: false,
+      truncate_ragged_lines: false,
```

Comment:
This should be `true`.

Reply:
Fixed! Just curious, do you know why it defaults to true only in this (internal) method?